src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -123,6 +123,54 @@ public boolean shouldTraverse(NodeTraversal nodeTraversal, Node n, Node parent)
   }
 
   /**
+   * This class detects the UMD pattern by checking if a node includes
+   * a "module.exports" or "exports" statement.
+   */
+  static class FindModuleExportStatements extends AbstractPreOrderCallback {
+
```

Comment:
Would `FindModuleExportStatements` be ok as a name instaed of `Detect....`. There is already an existing class called `FindGoogProvideOrGoogModule` (https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/ProcessCommonJSModules.java#L97) and I thought it would be ok if they would have slightly similar names.